index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+<head>
+    <script type="text/javascript" src="" async></script>
+    <link rel="stylesheet" type="text/css" href="">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.css">
+</head>
+
+<body>
+<div class="container">
+    <div class="menu">
+        <button class="instructionsButton"></button>
+        <button class="newGame"></button>
```

Comment:
Write 'New Game' between button tags

```suggestion
        <button class="newGame">New Game</button>
```